rpcs3/Emu/Cell/SPUASMJITRecompiler.cpp

```diff
@@ -2788,7 +2788,7 @@ void spu_recompiler::FSMB(spu_opcode_t op)
 		c->pshufd(va, va, 0xfa);
 	}
 
-	c->movdqa(vm, XmmConst(_mm_set_epi8(128, 64, 32, 16, 8, 4, 2, 1, 128, 64, 32, 16, 8, 4, 2, 1)));
+	c->movdqa(vm, XmmConst(_mm_set_epi8(-128, 64, 32, 16, 8, 4, 2, 1, -128, 64, 32, 16, 8, 4, 2, 1)));
```

Comment:
_mm_set_epu8 would be better instead, -128 is less understandable.

Reply:
Google gives 3 (three) results for this instruction and it crashes the travis build 😄 
Is there another include or something?

Reply:
My bad, I assumed there are unsigned versions for these macros.